pkg/printers/internalversion/printers.go

```diff
@@ -595,7 +595,7 @@ func formatEndpoints(endpoints *api.Endpoints, ports sets.String) string {
 	if len(endpoints.Subsets) == 0 {
 		return "<none>"
 	}
-	list := []string{}
+	var list []string
```

Comment:
I think they are the same.

The `:=` syntax is shorthand for declaring and initializing a variable

Reply:
Although the behavior is the same, it is not recommended